### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by Vrganj**
### Description
I updated 5 examples to work on 1.13.2+ (Probably lower versions, too)

---
**Target Minecraft Versions:**     any
**Requirements:**     No required plugins or Paper
**Related Issues:**     None

### CODE REVIEW

File: src/main/resources/scripts/equip anything.sk [hunk 1]
```diff
@@ -2,20 +2,20 @@
 # Allows specified items to be used as helmets for survival players (Minecraft 1.9 recommended).
 #
 
-aliases:
-	custom helmets = iron block, gold block, diamond block # Change this to anything you'd like
+init:
+	set {helmets::*} to iron block, gold block and diamond block
```
Comment (bensku) [inline src/main/resources/scripts/equip anything.sk]: Scriptt aliases should be preferred over random global variables. They're gone when the script is removed.

File: src/main/resources/scripts/kill counter.sk [hunk 2]
```diff
@@ -2,25 +2,20 @@
...
-	{kill counter.%player%.kills_total} = 0
-
-on death:
+death:
 	attacker is a player
-	add 1 to {kill counter.%attacker%.kills_total}
-	add 1 to {kill counter.%attacker%.kills_session}
-	{kill counter.%attacker%.kills_session} is greater than {kill counter.%attacker%.kills_session_max}:
-		set {kill counter.%attacker%.kills_session_max} to {kill counter.%attacker%.kills_session}
+	attacker isn't the victim
```
Comment (bensku) [inline src/main/resources/scripts/kill counter.sk]: Could you clarify when victim and attacker are the same in a comment?

File: src/main/resources/scripts/kill counter.sk [hunk 3]
```diff
@@ -2,25 +2,20 @@
...
 
-# Defaults are useful here, as a player's kills would be '<none>' if he hasn't killed anything yet,
-# but with defaults defined those will be used in that case.
-variables:
-	{kill counter.%player%.kills_session} = 0
-	{kill counter.%player%.kills_session_max} = 0
-	{kill counter.%player%.kills_total} = 0
-
-on death:
+death:
```
Comment (bensku) [inline src/main/resources/scripts/kill counter.sk]: Start trigger with 'on' for clarity.

File: src/main/resources/scripts/kill counter.sk [hunk 4]
```diff
@@ -2,25 +2,20 @@
...
 	attacker is a player
-	add 1 to {kill counter.%attacker%.kills_total}
-	add 1 to {kill counter.%attacker%.kills_session}
-	{kill counter.%attacker%.kills_session} is greater than {kill counter.%attacker%.kills_session_max}:
-		set {kill counter.%attacker%.kills_session_max} to {kill counter.%attacker%.kills_session}
+	attacker isn't the victim
+	add 1 to {kill counter::%attacker's uuid%::kills}
+	add 1 to {kill counter::%attacker's uuid%::streak}
+	{kill counter::%attacker's uuid%::streak} > {kill counter::%attacker's uuid%::max}:
```
Comment (bensku) [inline src/main/resources/scripts/kill counter.sk]: Why remove advanced variables section usage? Is it not working?

File: src/main/resources/scripts/kill counter.sk (hunk 2 above)
Comment (Vrganj) [reply src/main/resources/scripts/kill counter.sk]: Pretty sure it's when a player shoots themselves with a bow, happens sometimes.

File: src/main/resources/scripts/kill counter.sk (hunk 4 above)
Comment (Vrganj) [reply src/main/resources/scripts/kill counter.sk]: It's strange, but UUID's can't be used in the variables section.

File: src/main/resources/scripts/equip anything.sk (hunk 1 above)
Comment (Vrganj) [reply src/main/resources/scripts/equip anything.sk]: I tested those, acted strangely. They gave errors about how ID's can't be used in newer versions, even though I didn't use ID's.

File: src/main/resources/scripts/kill counter.sk (hunk 3 above)
Comment (Vrganj) [reply src/main/resources/scripts/kill counter.sk]: Yeah, I should do that. I got used to not using the `on`.

File: src/main/resources/scripts/equip anything.sk (hunk 1 above)
Comment (Nicofisi) [reply src/main/resources/scripts/equip anything.sk]: Could you send the code you used please?

File: src/main/resources/scripts/kill counter.sk (hunk 2 above)
Comment (Nicofisi) [reply src/main/resources/scripts/kill counter.sk]: *He probably meant to add that comment to the code*

Comment (Vrganj) [reply src/main/resources/scripts/kill counter.sk]: oh..

**Comment by ShaneBeee** [comment]
Im just going thru some older PRs
This one has been open nearly a year. Changes have been requested but these changes never happened.

I'm going to close this for now, but if changes are made in the future, feel free to re-open it.

**Pull request closed without merging**